core/src/main/java/org/ehcache/spi/cache/tiering/CachingTier.java

```diff
@@ -82,7 +82,7 @@
   interface Provider extends Service {
     <K, V> CachingTier<K, V> createCachingTier(Store.Configuration<K, V> storeConfig, ServiceConfiguration<?>... serviceConfigs);
 
-    void releaseCachingTier(CachingTier<?, ?> resource);
+    void releaseCachingTier(CachingTier<?, ?> resource) throws CacheAccessException;
```

Comment:
Same remark as above: do we want this exception to be rethrown or handled internally?